ui/js/reactrevisions.jsx

```diff
@@ -4,9 +4,9 @@ const MoreRevisionsLink = (props) => (
     <li>
         <a href={props.href}
            data-ignore-job-clear-on-click={true}
-           target='_blank'>
```

Comment:
It's not required, but double quotes are considered idiomatic for HTML/JSX. This comes along with the React preset.